src/uwtools/utils/tasks.py

```diff
@@ -46,13 +46,15 @@ def existing(path: Path):
 
 
 @external
-def file(path: Path):
+def file(path: Path, context: str = ""):
     """
     An existing file or symlink to an existing file.
 
     :param path: Path to the file.
+    :param context: Optional additional context for the file.
     """
-    yield "File %s" % path
+    suffix = " (" + context + ")" if context else ""
+    yield "File %s%s" % (path, suffix)
```

Comment:
An optional context parameter is added here to give some extra info about a file. The final key of the key path is given in the case of `chgres_cube` namelists, but this could be more complete key paths for other drivers or any other contextual clues for other applications.